packages/tools/grub/package.mk

```diff
@@ -9,6 +9,7 @@ PKG_LICENSE="GPLv3"
 PKG_SITE="https://www.gnu.org/software/grub/index.html"
 PKG_URL="http://git.savannah.gnu.org/cgit/grub.git/snapshot/$PKG_NAME-$PKG_VERSION.tar.gz"
 PKG_DEPENDS_TARGET="toolchain flex freetype:host gnulib:host gettext:host"
```

Comment:
Happy to say a `grub` rebuild works fine - even with `gnulib:host` dropped from `PKG_DEPENDS_TARGET`. `gnulib` is unpacked correctly by the recursive `PKG_DEPENDS_UNPACK` logic in `scripts/unpack`, so that's all good.

So, I think you just need to drop `gnulib:host`.

Reply:
Ok, will drop gnulib:host